.github/workflows/ci.yml

```
@@ -44,16 +44,76 @@ jobs:
+        with:
+          submodules: recursive
+
+      - uses: actions/setup-python@v5
         with:
           python-version: "3.12"
+
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+<<<<<<< HEAD
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)